packages/design-system/src/components/Promo/style.scss

```diff
@@ -81,6 +81,8 @@ $base-class: 'promo';
     }
     
     .#{$base-class}__header {
+      font-size: 16px;
```

Comment:
if the style is the same for every banner size you can declare it under `baseClass`. That way it will not be duplicated and the size of styles file will be smalller.

Reply:
done

Reply:
please move the whole `.#{$base-class}__header` under base class

Reply:
I removed margin, but I wouldn't remove the whole class. If i did it, children components will have bold font by default, that's a bad idea.

Reply:
Please take a look at https://github.com/livechat/design-system/pull/196. It seems that I can not describe it so I did a PR to this one with required changes.